Cargo.toml

```diff
@@ -14,13 +14,13 @@ panic = "abort"
 quicli = "0.4.0"
 csv = "1.1.3"
 indicatif = "0.15.0"
-strum = "0.18.0"
-strum_macros = "0.18.0"
-structopt = "0.3.15"
+strum = "0.19.2"
+strum_macros = "0.19.2"
+structopt = "0.3.18"
 # Use patched version until upstream fixed is merged.
 # https://github.com/tafia/quick-xml/pull/220
 quick-xml = { git = "https://github.com/pacman82/quick-xml.git" }
```

Comment:
This has been already changed on master, to the new upstream version. I suggest re-basing the PR.